synapse/storage/databases/main/registration.py

```diff
@@ -1526,34 +1526,48 @@ async def get_registration_tokens(
               Default is None: return all tokens regardless of validity.
 
         Returns:
-            A list of dicts, each containing details of a token.
+            A list of tuples containing:
+                * The token
+                * The number of users allowed (or None)
+                * Whether it is pending
+                * Whether it has been completed
```

Comment:
Return type says these are ints, presumably because they're bools in the DB and sqlite doesn't distinguish between bool and int etc. Can we convert them to ints here?

Reply:
Oh no, these genuinely are integers. I guess you can use a registration token more than once.

Reply:
I think you can mint a registration token w/ a certain number of uses.